src/uu/uptime/src/uptime.rs

```diff
@@ -11,17 +11,15 @@
 use chrono::{Local, TimeZone, Utc};
 use clap::{crate_version, Arg, ArgAction, Command};
 
-use uucore::format_usage;
+use uucore::{format_usage, help_about, help_usage};
 // import crate time from utmpx
 pub use uucore::libc;
```

Comment:
Entirely unrelated to this PR, but I wonder what this is for? I can't think of any reason this would need to be pub